modules/identity.bicep

```diff
@@ -21,12 +21,13 @@ module userAssignedIdentity 'managedIdentity/userAssignedIdentities.bicep' = {
 module appRoleAssignment 'authorization/appRoleAssignments.bicep' = {
   name: 'appRoleAssignment'
   scope: resourceGroup(_subscription, _appResourceGroupName)
```

Comment:
we could remove this, if we define explicitly define it in other file, like before